frontend/components/Comment.js

```diff
@@ -8,40 +8,76 @@ class Comment extends Component {
     this.setState({ editValue: event.target.value })
   }
   componentDidMount() {
+    let elems = document.querySelectorAll('.fixed-action-btn')
+    let instances = M.FloatingActionButton.init(elems, { direction: 'left' })
     this.setState((prevState) => ({
       editValue: this.props.content,
     }))
   }
   render() {
+    console.log(this.props)
     return (
       <React.Fragment>
         <div className="card">
+          <div
+            className="card-content grey lighten-4"
+            style={{
+              display: 'flex',
+              justifyContent: 'space-between',
+              alignItems: 'center',
+            }}
+          >
+            <div className="">
+              <span style={{ marginRight: 10 }}>
+                <b>{this.props.authorName}</b>
+              </span>
+              {new Date(this.props.createdAt).toDateString()}{' '}
+            </div>
+            <div
+              className="fixed-action-btn horizontal"
+              style={{ position: 'static' }}
+            >
+              {this.props.canEditOrDelete && (
+                <React.Fragment>
+                  <a className="btn-floating waves-effect waves-light ">
+                    <i className="large material-icons">more_vert</i>
+                  </a>
```

Comment:
Is this used? Maybe remove it if it has no purpose

Reply:
this one is used, this is the blue icon with :